introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-few-shot-learning.ipynb

```diff
@@ -0,0 +1,1896 @@
+{
```

Comment:
<p>There are plenty of other text generation models like <a href="https://huggingface.co/bigscience/bloom-7b1" target="_blank"><strong>Bloom 7b1</strong></a>, <a href="https://huggingface.co/bigscience/bloom-3b" target="_blank"><strong>Bloom 3b</strong></a>, <a href="https://huggingface.co/EleutherAI/gpt-neo-2.7B" target="_blank"><strong>GPT-NEO-2b7</strong></a>, and etc. --&gt; remove "plenty of"</p><p> </p><p>"make predictions thanks to only a couple of examples" needs rewriting.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>